server/api/themoviedb/index.ts

```diff
@@ -613,15 +613,15 @@ class TheMovieDb extends ExternalAPI {
 
         return tvshow;
       }
-
-      throw new Error(
-        `[TMDb] Failed to find a TV show with the provided TVDB ID: ${tvdbId}`
-      );
     } catch (e) {
       throw new Error(
         `[TMDb] Failed to get TV show using the external TVDB ID: ${e.message}`
       );
     }
+
+    throw new Error(
+      `[TMDb] Failed to find a TV show with the provided TVDB ID: ${tvdbId}`
+    );
```

Comment:
Not sure what the point of moving the error down here does? Instead why not change the thrown error message on lines 616 to say something like `No show returned from API for ID <id>`  or something along those lines? Making the full message read `[TMDb] Failed to get TV show using the external TVDB ID: No show returned from API for ID <id>`

Reply:
Yeah, sure!  I just wanted to remove the redundant part.